constants/pokemon_constants.asm

```diff
@@ -272,9 +272,8 @@ JOHTO_POKEMON EQU const_value
 	const HO_OH      ; fa
 	const CELEBI     ; fb
 NUM_POKEMON EQU const_value - 1
-	const MON_FC     ; fc
+	const_skip       ; fc
 	const EGG        ; fd
-	const MON_FE     ; fe
```

Comment:
Doesn't this one have palette slots/entries in other tables? If so, keep it with the name.

Reply:
Some tables stop at Celebi; some have data for 252, Egg, 254, and also 255 (which is not a valid `MON_*` ID). How about putting back `MON_FE` and adding `CANCEL`?

Reply:
CANCEL is a universal menu-related constant, not a pokemon. While I think it's a worthwhile addition, it shouldn't be here. I guess the extra entries aren't necessary if some tables stop at different places, anyway.